test/lib/y2users/linux/writer_test.rb

```diff
@@ -100,6 +103,21 @@
       allow(Yast::Execute).to receive(:on_target!)
     end
 
+    context "for an existing user" do
+      before do
+        initial_config.users << user
+        initial_config.passwords << password
```

Comment:
this collide with https://github.com/yast/yast-users/pull/254. We would need to update one or another PR. ( just note )